server/src/main/java/com/cloud/network/IpAddressManagerImpl.java

```diff
@@ -732,8 +732,9 @@ public boolean disassociatePublicIpAddress(long addrId, long userId, Account cal
         PublicIpQuarantine publicIpQuarantine = null;
         // Cleanup all ip address resources - PF/LB/Static nat rules
         if (!cleanupIpResources(addrId, userId, caller)) {
```

Comment:
Are the IP resources that are cleaned up by the `cleanupIpResources` method being rolled back when the exception is thrown?

Reply:
@weizhouapache can you check this ^^^